tsconfig.json

```diff
@@ -21,7 +21,7 @@
     "next-env.d.ts",
     "**/*.ts",
     "**/*.tsx"
-  ],
+, "pages/_app.js"  ],
```

Comment:
I might need you to explain what's going on here. Are we overriding the default behavior so that we can add a authentication layer?

Reply:
1. In the original commit the change of _app by Jeff to .js to .ts appeared as a delete / create.
2. I then reverted that change (.ts to .js), then used git mv to change the file (.js back to .ts) so there would be history. This appears to be an artifact of that change when references were automatically updated.

I'll fix that now.